sdk/samples/iot/sample_pnp.c

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "iot_sample_common.h"
 #include "sample_pnp.h"
+#include "iot_sample_common.h"
```

Comment:
Looks like this header order matters and re-ordering it is causing CI failures. @momuno - this is something we should fix-up, either by adding the necessary header files OR adding a comment and putting a new line between the include blocks that need to go in the certain order.

VS auto-format it and ordered it in some alpha order.

Reply:
i'll take a look at it. thanks.